src/test/java/org/opensrp/repository/postgres/ClientsRepositoryTest.java

```diff
@@ -512,7 +512,34 @@ public void shouldGetMemberCountHouseholdHeadProviderByClients() {
 		id.add("28caef27-d1b3-497b-8a55-95Rf2f0f6e24");
 		List<HouseholdClient> householdClient = clientsRepository.selectMemberCountHouseholdHeadProviderByClients("", id,
 		    "ec_family");
-		assertEquals(householdClient.size(), 0);
 		assertNotNull(householdClient);
 	}
+	
+	@Test
+	public void shouldFindAllClientsByCriteria() {
+		AddressSearchBean addressSearchBean = new AddressSearchBean();
+		ClientSearchBean searchBean = new ClientSearchBean();
+		searchBean.setClientType("ec_family");
+		List<Client> clients = clientsRepository.findAllClientsByCriteria(searchBean, addressSearchBean);
+		assertNotNull(clients);
+	}
+	
+	@Test
+	public void shouldFindCountAllClientsByCriteria() {
+		AddressSearchBean addressSearchBean = new AddressSearchBean();
+		ClientSearchBean searchBean = new ClientSearchBean();
+		searchBean.setClientType("ec_family");
+		HouseholdClient clientCustomField = clientsRepository.findCountAllClientsByCriteria(searchBean, addressSearchBean);
+		assertNotNull(clientCustomField);
+		assertNotEquals(clientCustomField.getTotalCount(), 0);
```

Comment:
For asserts the expected value should always be on the left side . e.g.
assertNotEquals(0, clientCustomField.getTotalCount());